podinfo crashes with panic when the WebSocket endpoint is under load

Hi there, first of all, thank you very much for providing this project and keeping it active.

I was using podinfo as target for some WebSocket related load testing, and I have received some random panics when under having load as following:

```
panic: send on closed channel

goroutine 6574 [running]:
github.com/stefanprodan/podinfo/pkg/api/http.(*Server).sendHostWs(0x40000305f0, 0x89154?, 0x400685e4e0, 0x400685e300)
	/podinfo/pkg/api/http/echows.go:70 +0xcc
created by github.com/stefanprodan/podinfo/pkg/api/http.(*Server).echoWsHandler in goroutine 6582
	/podinfo/pkg/api/http/echows.go:37 +0x29c
```

which means that in some edge cases, the timer in the code below, is trying to write after the channel was already closed:
https://github.com/stefanprodan/podinfo/blob/f3506240479e9263db500e67d8bb8908e52fc2d1/pkg/api/http/echows.go#L57-L76

--- a/pkg/api/http/echows.go
+++ b/pkg/api/http/echows.go
@@ -3,6 +3,7 @@ package http
 import (
 	"net/http"
 	"strings"
+	"sync"
 	"time"
 
 	"github.com/gorilla/websocket"
@@ -28,13 +29,19 @@ func (s *Server) echoWsHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	var wg sync.WaitGroup
+	wg.Add(1)
+
 	defer c.Close()
 	done := make(chan struct{})
 	defer close(done)
 	in := make(chan interface{})
-	defer close(in)
 	go s.writeWs(c, in)
-	go s.sendHostWs(c, in, done)
+	go s.sendHostWs(c, in, done, &wg)
+	go func() {
+		defer close(in)
+		wg.Wait()
+	}()
 	for {
 		_, message, err := c.ReadMessage()
 		if err != nil {
@@ -54,7 +61,7 @@ func (s *Server) echoWsHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func (s *Server) sendHostWs(ws *websocket.Conn, in chan interface{}, done chan struct{}) {
+func (s *Server) sendHostWs(ws *websocket.Conn, in chan interface{}, done chan struct{}, wg *sync.WaitGroup) {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 	for {
@@ -70,6 +77,7 @@ func (s *Server) sendHostWs(ws *websocket.Conn, in chan interface{}, done chan s
 			in <- status
 		case <-done:
 			s.logger.Debug("websocket exit")
+			wg.Done()
 			return
 		}
 	}

